src/arbitraries/Arbitrary.ts

```diff
@@ -44,7 +46,34 @@ export abstract class Arbitrary<A> {
   }
 
   /**
-   * The special cases for this arbitrary, which can be used during sampling to give higher weight to certain elements.
+   * Returns a sample of picks of a given size without replacements.
+   *
+   * While this function doesn't pick corner cases, each pick is not necessarily independent
```

Comment:
This description does not look right given that we actually require all elements to be unique here. Can you update it accordingly?